docs/detections/invest-guide-run-osquery.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[invest-guide-run-osquery]]
+=== Run Osquery from investigation guides
+Investigations guides can be used with rules and provide guidance for triaging, analyzing, and responding to potential security issues. When you build a custom rule, you can also set up an investigation guide that incorporates Osquery. This allows you to run live queries from a rule's investigation guide as you analyze alerts produced by the rule.
+
+[role="screenshot"]
+image::images/osquery-investigation-guide.png[Shows a live query in an investigation guide]
+
+
+.Requirements
```

Comment:
Maybe move the reqs above the screenshot -- they kind of get buried under the image?